src/services/fileStorage/strategies/awsS3.js

```diff
@@ -335,6 +335,10 @@ class AWSS3Strategy extends AbstractFileStorageStrategy {
 		};
 	}
 
+	getBucket(schoolId) {
```

Comment:
why adding this to the existing getBucketName() ?

Reply:
`getBucketName` will not be exported. In contrast to `getBucketName`, `getBucket` is part of the strategy. Currently only S3 storage providers are supported, but for other storage provider the bucket name pattern could differ.